package.json

```diff
@@ -70,23 +75,54 @@
     "@electron-forge/maker-zip": "6.0.0-beta.50",
     "@electron-forge/plugin-webpack": "6.0.0-beta.50",
     "@marshallofsound/webpack-asset-relocator-loader": "^0.5.0",
+    "@types/cucumber": "^6.0.1",
+    "@types/enzyme": "^3.10.4",
+    "@types/enzyme-adapter-react-16": "^1.0.5",
+    "@types/jest": "^25.2.1",
+    "@types/node": "^13.13.4",
     "@typescript-eslint/eslint-plugin": "^2.18.0",
     "@typescript-eslint/parser": "^2.18.0",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
     "css-loader": "^3.0.0",
+    "cucumber": "^6.0.5",
+    "cucumber-pretty": "^6.0.0",
+    "cucumber-tsflow": "^3.2.0",
     "electron": "8.2.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-to-json": "^3.4.4",
     "eslint": "^6.8.0",
     "eslint-plugin-import": "^2.20.0",
     "fork-ts-checker-webpack-plugin": "^3.1.1",
+    "jest": "^25.3.0",
+    "jest-cli": "^25.3.0",
+    "jest-cucumber": "^2.0.11",
```

Comment:
Do we need this one here?

Reply:
We need jest and jest-cli, but don't need jest-cucumber so I delete it